tests/test_paramviewer.py

```diff
@@ -10,14 +10,14 @@ def test_paramviewer_simple_nonbatched(backend):
     view = ParamViewer(
         parshape,
         {'hello': {'slice': slice(0, 2)}, 'world': {'slice': slice(5, 7)}},
-        ['hello', 'world'],
+        ['world', 'hello'],
     )
     par_slice = view.get(pars)
-    assert pyhf.tensorlib.tolist(par_slice[view.slices[0]]) == [1, 2]
+    assert pyhf.tensorlib.tolist(par_slice[slice(2, 4)]) == [1, 2]
 
-    assert pyhf.tensorlib.tolist(par_slice[view.slices[1]]) == [6, 7]
+    assert pyhf.tensorlib.tolist(par_slice[slice(0, 2)]) == [6, 7]
 
-    assert pyhf.tensorlib.tolist(par_slice) == [1, 2, 6, 7]
+    assert pyhf.tensorlib.tolist(par_slice) == [6, 7, 1, 2]
```

Comment:
As the choice in ordering here results in changes beyond just `view.slices` -> `slice` is there a pedagogical, style, or preference for doing so? If so that's fine, just curious. :)

Reply:
there is a feature in paramviewers that does some ordering, and the change in the test is explicitly chosen now to make clear that the ordering happens